packages/admin-portal/src/components/pages/subscriptions/index.tsx

```diff
@@ -0,0 +1,166 @@
+) => {
+  const query = setQueryParams(queryParams)
+  let queryString = `?page=${page}`
+  if (query && query !== '') {
+    queryString = queryString.concat(`&${query}`)
+  }
+  return history.push(`${Routes.SUBSCRIPTIONS}${queryString}`)
+}
+
+export const onSearchHandler = (history: History<any>) => (
+  queryParams: SubscriptionsFilterFormValues,
+  { setStatus },
+) => {
+  const cleanedValues = cleanObject(queryParams)
+
+  if (isEmptyObject(cleanedValues)) {
+    setStatus('Please enter at least one search criteria')
+    return
+  }
+
+  const query = setQueryParams(cleanedValues)
+  if (query && query !== '') {
+    const queryString = `?page=1&${query}`
+    history.push(`${Routes.SUBSCRIPTIONS}${queryString}`)
+  }
+}
+
+const Subscriptions: React.FC = () => {
+  const history = useHistory()
+  const location = useLocation()
+  const search = location.search
+  const queryParams = new URLSearchParams(location.search)
+  const filterValues = buildFilterValues(queryParams)
+  const onPageChange = React.useCallback(onPageChangeHandler(history, filterValues), [history, filterValues])
+  const onSearch = React.useCallback(onSearchHandler(history), [history])
+
+  const SubscriptionsListState = useSelector(selectSubscriptionListState)
+
+  const page = queryParams.get('page') ? Number(queryParams.get('page')) : 1
+  useEffect(() => {
+    if (search) {
+      store.dispatch(fetchSubscriptionList({ page, queryString: search } as FetchSubscriptionListQuery))
+    }
+  }, [search])
+
+  const [cancelSubId, setCancelSubId] = React.useState<string>('')
+
+  const onCancelSubscription = () => {
+    store.dispatch(cancelSubscription({ id: cancelSubId } as CancelSubscriptionParams))
+  }
+
+  const { data, totalCount, pageSize, pageNumber = 1, isLoading } = SubscriptionsListState
+
+  const CreatedCell = ({ cell: { value } }) => <p>{toLocalTime(value)}</p>
+
+  const StatusCell = ({ cell: { value } }) => <p>{value ? 'Deactived' : 'Active'}</p>
+
+  const DeleteButton = ({ cell: { value } }) => (
+    <Button type="button" variant="danger" onClick={() => setCancelSubId(value)}>
+      Cancel
+    </Button>
+  )
+
+  const columns = [
+    { Header: 'Subscription Type', accessor: 'type' },
+    { Header: 'Summary', accessor: 'summary' },
+    { Header: 'Application ID', accessor: 'applicationId' },
+    { Header: 'Member Name', accessor: 'developerId', Cell: MemberNameCell },
+    { Header: 'Member Email', accessor: 'user' },
+    {
+      Header: 'Start Date',
+      accessor: 'created',
+      Cell: CreatedCell,
+    },
+    {
+      Header: 'Renewal Date',
+      accessor: 'renews',
+    },
+    { Header: 'Frequency', accessor: 'frequency' },
+    { Header: 'Cost', accessor: 'cost' },
+    { Header: 'Status', accessor: 'cancelled', Cell: StatusCell },
+    {
+      Header: '',
+      accessor: 'id',
+      Cell: DeleteButton,
+    },
+  ]
+
+  return (
+    <ErrorBoundary>
+      <Section className="mb-0">
+        <H3>Subscriptions</H3>
```

Comment:
You can just use the H3 not wrapped in a section by using the `isHeadingSection` prop eg `<H3 isHeadingSection>...`

Reply:
okay, will check it